pytorch_lightning/trainer/callback_hook.py

```diff
@@ -233,3 +265,10 @@ def on_before_zero_grad(self, optimizer):
         """
         for callback in self.callbacks:
             callback.on_before_zero_grad(self, self.lightning_module, optimizer)
+
+    @staticmethod
+    def _is_param_in_hook_signature(hook_fx: Callable, param: str) -> bool:
```

Comment:
Can you add support for users who use `*args` in their signature?

Reply:
What do you mean?

Reply:
some people do:

```python
def on_test_epoch_end(self, *args):
    ...

# or

def on_test_epoch_end(self, trainer, *args):
    ...
```

Those cases should not raise a Warning

Reply:
Ahh, yeah! Updated.